apex-app-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -50,6 +50,12 @@
       </includes>
     </fileSet>
     <fileSet filtered="true" encoding="UTF-8">
+      <directory>src/main/resources</directory>
+      <includes>
+        <include>**/log4j.props</include>
```

Comment:
Hmmm, I would have still preferred to use log4j.properties as that is the standard. But looking at the issues you faced I am okay to use apexlog4j.properties.